app/src/main/java/org/oppia/android/app/databinding/DrawableBindingAdapters.java

```diff
@@ -36,4 +36,14 @@ public static void setTopBackgroundDrawable(@NonNull View view, @ColorInt int co
     GradientDrawable background = (GradientDrawable) view.getBackground();
     background.setColor(0xff000000 | colorRgb);
   }
+
+  /** Used to set a rounded-rect background drawable with a data-bound color. */
```

Comment:
To differentiate it from the other smaller round-rect documentation and make its intended purpose more evident.
```suggestion
  /** Used to set a large rounded-rect background drawable with a data-bound color. */
```

Reply:
updated